pymodaq_plugins/hardware/keithley2110/keithley2110_VISADriver.py

```diff
@@ -1,4 +1,18 @@
-import pyvisa as visa
```

Comment:
Why not adding this driver checking in a visa specific folder within hardware. It may be useful for many other plugins.

Reply:
You mean keithley plugin should call a hardware/visa driver ? ok I will do that

Reply:
yes that would be used by all plugins using visa as a protocol

Reply:
why not ? 
maybe it's a good idea to use [instbuilder](https://lucask07.github.io/instrbuilder/build/html/index.html) ?

Reply:
I'm not sure it would provide enough flexibility (and some more knowledge is needed for this interface). Keeping things simple, you should wrap in your visa driver, things to list devices (listDevices as a class method of the driver for instance), opening the ressource in a method...
The specific visa instrument wrapper you work on could then subclass from this and just focus on specific commands. What do you think?